password-update-service-samples/README.md

```diff
@@ -0,0 +1,21 @@
+# Password Update Service Samples
+
+This repository contains sample implementations of password update services for [Asgardeo](https://console.asgardeo.io/) and [WSO2 Identity Server](https://wso2.com/identity-server/). These samples demonstrate how to create and configure a password update mechanism as an external service extension to the product.
```

Comment:
Let's be specific on what this sample is about rather than generic on that this is a password update sample.
Look at https://github.com/asgardeo-samples/asgardeo-service-extension-samples/blob/main/custom-authentication-service-samples/pin-based-authentication-service-express/README.md

This clearly says that this is a pin based authenticator, similarly mention that this is about compromised password validation with haveibeenpwned

Follow the naming convention used in above
e.g
pin-based-authentication-service-express
`<what the extension is about>-<technology>`